quickstart.sh

```diff
@@ -54,6 +56,10 @@ do
     --online-deployment | -online)
       local_deploy=false
       echo -e "${MAJOR}Configuring Chorus for chorus-opensearch-edition.dev.o19s.com environment\n${RESET}"
+      ;;
```

Comment:
is there some white space formatting needed to align with whats above?   Or maybe just how github renders it in the web view?

Reply:
good catch, didn't see that in the IDE.

Other options were broken as well, tried to align what's there.